geomesa-gs-plugin/geomesa-accumulo-gs-plugin/README.md

```diff
@@ -1,11 +1,33 @@
-# GeoMesa Plugin
+# GeoMesa Accumulo GeoServer Plugin
 
-### Building Instructions
+### Installation Instructions
```

Comment:
looks like jai is already included in geoserver. Vecmath appears to only be pulled in by geomesa-kafka as a test dependency... do you know if it's needed?